contribution.md

```diff
@@ -0,0 +1,81 @@
+# Welcome to TCET Open Source contributing guide <!-- omit in toc -->
+
+Thank you for investing your time in contributing to our project! Any contribution you make will be reflected on [Contributors](contributors.md).
+
+Read our [Code of Conduct](./CODE_OF_CONDUCT.md) to keep our community approachable and respectable.
```

Comment:
You can comment these out for now, or remove them, since we dont actually have these yet.